### DISCUSSION TITLE
Filter Database Objects does not work for Teradata Connection/DB

### DISCUSSION THREAD

**Discussion by ahhmedbilal** [dbeaver/dbeaver#13205] [Q&A]
#### System information: 
- Operating system (distribution) and version: Win 10
- DBeaver version: 21.1.0.202106012023
- Additional extensions

#### Connection specification:
- Database name and version: Teradata 16.20
- Driver name: Teradata
jdbc:teradata://hostname.com/DATABASE=DBC,DBS_PORT=1025,LOGMECH=TDNEGO,ENCRYPTDATA=ON,CHARSET=UTF8,SIP_SUPPORT=OFF
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
On "edit connection" box -> general -> filter: database/catalog -> add filter as PD% but still it shows all databases.

**Comment by ahhmedbilal** [comment]
Apart from above issue: in database navigator, if i add PD in filter box, then it shows all database (table container) starting with PD, but if i write PD|PV 
or if i write PD,PV then nothing is shown....

**Comment by LonwoLonwo** [comment]
Hello @ahhmedbilal 
About your second message:
Support of muli-filtering was added only in DBeaver 21.1.1 https://github.com/dbeaver/dbeaver/issues/10137
It doesn't work in DBeaver 21.1.0

**Comment by LonwoLonwo** [comment]
Oh, I see it. The problem is in database/schema designation misunderstanding. Try to use "Schemas/Users" filtering instead of "Databases/Catalogs".

![2021-07-15 11_32_48-Window](https://user-images.githubusercontent.com/45152336/125756878-fcb547d0-afaf-4642-bb98-73d933de0ebc.png)

**Comment by ahhmedbilal** [comment]
yup it worked perfectly.

**Comment by LonwoLonwo** [comment]
Thanks for the answer!
